se/spelling.py

```diff
@@ -119,6 +119,9 @@ def detect_problem_spellings(xhtml: str) -> list:
 	if regex.search(r"\bfree[\-\s]?will", xhtml):
 		output.append("“freewill” or “free will” or “free-will” detected. Confirm that “free will” and “free-will” are strictly nouns, and that “freewill” is strictly an adjective.")
 
+	if regex.search(r"\bna[iï]f", xhtml):
```

Comment:
It occurs to me that we should have a case-insensitive flag here, and also added to the previous checks too

Reply:
Good plan, added.